awesome_cart/templates/pages/cart.py

```diff
@@ -37,6 +37,13 @@ def get_context(context):
 			addresses.extend(frappe.get_all("Address", filters={"name" : address.parent, "disabled" : False}, fields="*"))
 		context['addresses'] = addresses
 
+	if frappe.session.user != "Guest":
+		customer_info = frappe.db.get_values("Customer", get_current_customer().name, ["has_shipping_account", "fedex_account_number"])[0]
```

Comment:
Will this work if there is no "Customer" for that "User"?

Reply:
If a User doesn't have a Customer, they won't have the option to use that feature

Reply:
No, but then this line will break. Cause you're expecting a list in return.